hnn_core/viz.py

```diff
@@ -1218,6 +1218,7 @@ def plot_laminar_csd(times, data, contact_labels, ax=None, colorbar=True,
 
     ax.set_xlabel('Time (ms)')
     ax.set_ylabel('Electrode depth')
+    ax.set_xlim(right=times[-1])
```

Comment:
This is a nitpick, but I'd recommend defining `tstop` (or whatever you want to call it) before decimating so that you don't introduce a small error in the right x-limit.

Reply:
Ah you're right. I didn't realize that [`_get_plot_data_trange`](https://github.com/tianqi-cheng/hnn-core/blob/d2c5c8c506dcbde33856f0fa3519674ab4a391b3/hnn_core/viz.py#L24) only crops the time series. Since this is the case, however, I don't think we need the added lines on L1207-1209 as they crop the time series to its original length (i.e., they don't actually crop anything off).

Reply:
Re: adding the decimate feature to `plot_laminar_csd`, my vote would be to save it for a separate PR.